warehouse/upload.go

```diff
@@ -1642,6 +1642,7 @@ func (job *UploadJob) setUploadError(statusError error, state string) (string, e
 	}
 
 	serializedErr, _ := json.Marshal(&uploadErrors)
+	serializedErr = warehouseutils.SanitizeJson(serializedErr)
```

Comment:
We were getting this error while updating:
`unable to change upload columns: pq: unsupported Unicode escape sequence`